--- installwatch/installwatch.c	2008-11-16 22:20:53.000000000 +0600
+++ installwatch/installwatch_new.c	2016-05-06 21:30:03.915684951 +0600
@@ -100,7 +100,7 @@
 static int (*true_lxstat)(int,const char *,struct stat *);
 static int (*true_scandir)(	const char *,struct dirent ***,
 				int (*)(const struct dirent *),
-				int (*)(const void *,const void *));
+				int (*)(const struct dirent **,const struct dirent **));
 static int (*true_symlink)(const char *, const char *);
 static int (*true_truncate)(const char *, TRUNCATE_T);
 static int (*true_unlink)(const char *);
@@ -120,7 +120,7 @@
 static struct dirent64 *(*true_readdir64)(DIR *dir);
 static int (*true_scandir64)(	const char *,struct dirent64 ***,
 				int (*)(const struct dirent64 *),
-				int (*)(const void *,const void *));
+				int (*)(const struct dirent64 **,const struct dirent64 **));
 static int (*true_xstat64)(int,const char *, struct stat64 *);
 static int (*true_lxstat64)(int,const char *, struct stat64 *);
 static int (*true_truncate64)(const char *, __off64_t);
@@ -2938,7 +2938,7 @@
 	return result;
 }
 
-#if (GLIBC_MINOR <= 4)
+#if (0)
 int readlink(const char *path,char *buf,size_t bufsiz) {
 	int result;
 #else
@@ -3079,7 +3079,7 @@
 
 int scandir(	const char *dir,struct dirent ***namelist,
 		int (*select)(const struct dirent *),
-		int (*compar)(const void *,const void *)	) {
+		int (*compar)(const struct dirent **,const struct dirent **) ) {
 	int result;
 
 	if (!libc_handle)
@@ -3691,7 +3691,7 @@
 
 int scandir64(	const char *dir,struct dirent64 ***namelist,
 		int (*select)(const struct dirent64 *),
-		int (*compar)(const void *,const void *)	) {
+		int (*compar)(const struct dirent64 **,const struct dirent64 **)	) {
 	int result;
 
 	if (!libc_handle)
